webpack.config.js

```diff
@@ -0,0 +1,65 @@
+const webpack = require('webpack');
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+  context: path.resolve(__dirname, 'examples/src'),
+  entry: {
+    app: './app.js',
+  },
+  output: {
+    path: path.resolve(__dirname, 'examples/dist'),
+    filename: '[name].js',
+    publicPath: '/',
+  },
+  devServer: {
+    contentBase: path.resolve(__dirname, 'examples/src'),
+    port: 8080,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: [/node_modules/],
+        use: [{
+          loader: 'babel-loader',
+          options: { presets: ['es2015', 'stage-0', 'react'] },
```

Comment:
Same as comment from rollup config. Let's just use the `babelrc` config since there is no real difference.

Reply:
yup, done 👍